README.md

```diff
@@ -12,7 +12,8 @@ Variant's basic building blocks are:
 - `variant<Ts...>` - a type-safe representation for sum-types / discriminated unions
 - `recursive_wrapper<T>` - a helper type to represent recursive "tree-like" variants
 - `apply_visitor(visitor, myVariant)` - to invoke a custom visitor on the variant's underlying type
-- `get<T>()` - a function to directly unwrap a variant's underlying type
+- `bool is<T>()` - a function to test for a specific underlying type `T`
+- `T get<T>()` - a function to directly unwrap a variant's underlying type
```

Comment:
My two cents. I don't think `is<T>()` is so basic to stay here. The section is about basic building blocks. I believe this is more interesting to `mapox/variant.hpp`.